django/core/files/move.py

```diff
@@ -66,7 +66,13 @@ def file_move_safe(old_file_name, new_file_name, chunk_size=1024 * 64, allow_ove
         finally:
             locks.unlock(fd)
             os.close(fd)
-    copystat(old_file_name, new_file_name)
+
+    try:
+        copystat(old_file_name, new_file_name)
+    except PermissionError as e:
```

Comment:
Please use `PermissionError` for this PR and I'll change it when backporting to 1.11.